### DISCUSSION TITLE
Bing webmaster tools integration

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#99] [Anything Else]
"Create integration with Bing Webmaster Tools to get access to the search terms used to find your website. Bing Webmaster Tools gives access to search terms from both Bing and Yahoo! search. Unfortunately it looks like other Bing-based search engines are not included here (DuckDuckGo, Ecosia, etc.)"

([imported from the old roadmap](https://plausible.nolt.io/26))

**Comment by kolosek** [comment]
@metmarkosaric Hi Marko. Is there an update regarding bing webmaster tools integration?

**Reply by metmarkosaric** [reply]
There's no update at this time unfortunately. It's one of the many requests we have and is not [on our current roadmap](https://github.com/plausible/analytics/projects/1). We may add it at some point in the future but there are no promises. As usual if someone contributes to making this possible, it will get there faster.

**Comment by Swiss-Mac-User** [comment]
As Bing is getting way more popular through recent AI buzz (seen in the traffic sources where bing rises), I would love to have this for keyword analytics from bing queries 🙏

The API access to Bing‘s webmaster tools data are documented here: https://learn.microsoft.com/en-gb/bingwebmaster/
